src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -628,6 +642,18 @@ export default class BrowserCell extends Component {
             onPointerCmdClick(value);
           } else {
             setCopyableValue(hidden ? undefined : this.copyableValue);
+            if (selectedObjectId !== this.props.objectId) {
```

Comment:
I wasn't able to find the cause, but no matter if you are in a collection with/without data panel setup, changing the row results in a request to dashboard path. I guess the `Sidebar` component started to be rendered more often.
It's an unnecessary request and render, and in a long run it might affect the performance.

![request on row change](https://github.com/user-attachments/assets/50bf65d0-c4af-4440-8151-4aeedf8bfb9f)

Reply:
Thanks @vardhan0604 for the fix, it's better now. The request is no longer send while changing the rows with the keyboard, but it is when clicking with the mouse. So the steps are:
1. Open any collection
2. Click on any cell
3. Click on another cell from the different row

Repeating the step three results in a request being sent (watch the debugger, network tab). Such behavior is not present on the `alpha` branch. Do you think that can be fixed too?